sportnet/sportnet/authapp/forms.py

```diff
@@ -0,0 +1,38 @@
+import hashlib
+import random
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import AuthenticationForm, UserCreationForm
+
+from sportnet.sportnet.authapp import models
+
+User = get_user_model()
+
+
+class UserLoginForm(AuthenticationForm):
+    class Meta:
+        fields = ('username', 'password')
+
+    def __init__(self, request=None, *args, **kwargs):
+        super(UserLoginForm, self).__init__(request, *args, **kwargs)
+        for field_name, field in self.fields.items():
+            field.widget.attrs['class'] = 'uk-form-horizontal'
+
+
+class UserRegisterForm(UserCreationForm):
+    class Meta:
+        model = models.User
```

Comment:
Ты же импортировал пользователя отдельно, черезget_user_model, это здесь лишнее
```suggestion
        model = User
```